### PR TITLE
Changes R2_MODE and TEXT_ALIGN_OPTIONS

### PR DESCRIPTION
**Pull request by elachlan**
Fixes: 
- https://github.com/microsoft/win32metadata/issues/1036
- https://github.com/microsoft/win32metadata/issues/1045
```
Windows.Win32.Graphics.Gdi.TEXT_ALIGN_OPTIONS :  => [Flags]
Windows.Win32.Graphics.Gdi.Apis.GetROP2 : return...Int32 => R2_MODE
Windows.Win32.Graphics.Gdi.Apis.GetTextAlign : return...UInt32 => TEXT_ALIGN_OPTIONS
```

### CODE REVIEW

**Comment by elachlan** [comment]
CC: @lonitra

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -3861,6 +3861,7 @@
     "type": "uint",
     "namespace": "Windows.Win32.Graphics.Gdi",
     "name": "TEXT_ALIGN_OPTIONS",
```
Comment (elachlan) [inline generation/WinSDK/enums.json:3863]: Should `TEXT_ALIGN_OPTIONS` use `autoPopulate`? Would that mean we would only need to define `TA_MASK` in `members`?

Comment (sotteson1) [reply generation/WinSDK/enums.json:3863]: Yes, you could do that, but I would only change it if something is broken. If it's already working, I would leave it as is.

Comment (elachlan) [reply generation/WinSDK/enums.json:3863]: I will leave it as is then.

**Comment by elachlan** [comment]
Cc: @AArnott

**Comment by sotteson1** [comment]
The new system gets rid of the baseline .winmd, and instead updates a text file that will include the diff of your changes. (The output of the build is meant to help you automate your additions to this file.) Sorry to randomize your PR but I think this system will be a lot nicer for merges in the future.

**Review by riverar — APPROVED** [review]

**Review by sotteson1 — APPROVED** [review]
Thanks for all the contributions you've been giving us!

**Pull request merged**